common/src/app/common/types/tokens_lib.cljc

```diff
@@ -1445,20 +1450,35 @@ Will return a value that matches this schema:
         ;; Active themes without exposing hidden penpot theme
         active-themes-clear
         (-> (get-active-theme-paths tokens-lib)
-            (disj hidden-theme-path))
+            (disj hidden-theme-path))]
+    {:themes themes
+     :active-themes active-themes-clear}))
 
-        update-token-fn
-        (fn [token]
-          (cond-> {"$value" (:value token)
-                   "$type" (cto/token-type->dtcg-token-type (:type token))}
-            (:description token) (assoc "$description" (:description token))))
+(defn export-dtcg-multi-file
+  "Convert a TokensLib into a plain clojure map, suitable to be encoded as a multi json files each encoded in DTCG format."
+  [tokens-lib]
+  (let [{:keys [themes active-themes]} (dtcg-export-themes tokens-lib)
+        sets (->> (get-sets tokens-lib)
+                  (map (fn [{:keys [name tokens]}]
+                         [(str name ".json") (tokens-tree tokens :update-token-fn token->dtcg-token)]))
+                  (into {}))]
+    (-> sets
+        (assoc "$themes.json" themes)
+        (assoc-in ["$metadata.json" "tokenSetOrder"] (get-ordered-set-names tokens-lib))
+        (assoc-in ["$metadata.json" "activeThemes"] active-themes)
+        (assoc-in ["$metadata.json" "activeSets"] (get-active-themes-set-names tokens-lib)))))
```

Comment:
I think this can be
```clojure
(-> (update "$metadata.json"
      (assoc "tokenSetOrder" (get-ordered...)
             "activeThemes" active-themes
             "activeSets (get-active...)))
```